src/lib.rs

```diff
@@ -2024,6 +2025,19 @@ pub mod string {
         type Container = Strings;
     }
 
+    impl Columnar for Box<str> {
+        #[inline(always)]
+        fn copy_from<'a>(&mut self, other: crate::Ref<'a, Self>) {
+            let mut s = String::from(std::mem::take(self));
+            s.clear();
+            s.push_str(other);
+            *self = s.into_boxed_str();
```

Comment:
This potentially mitigates the benefits of `copy_from` because `Box<str>` cannot handle surplus capacity like `String` can. This will cause reallocations unless the strings have equal length.

Reply:
Well, it's better than the default impl in the case that they have the same length. :D